src/components/mainView/header.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Stack } from "react-bootstrap";
+
+interface IHeaderProps {
+  setShow: (x: boolean) => void;
+}
+const Header: React.FC<IHeaderProps> = (props) => {
+  return (
+    <>
+      <div className="mainHdg">Student Portal</div>
+      <hr style={{ borderColor: "gray" }} />
+      <Stack
+        direction="horizontal"
+        style={{ justifyContent: "space-between", margin: "20px" }}
+      >
+        <div>
```

Comment:
this is useless

Reply:
i've used it to seperate the **Add** button and the **Student Summary** text to the opposites sides of the container

Reply:
I meant this div seems to be useless